app/models/fhir/library.rb

```diff
@@ -2,177 +2,194 @@ module FHIR
-    embeds_one :approvalDate, class_name: 'FHIR::PrimitiveDate'    
-    embeds_one :lastReviewDate, class_name: 'FHIR::PrimitiveDate'    
-    embeds_one :effectivePeriod, class_name: 'FHIR::Period'    
-    embeds_many :topic, class_name: 'FHIR::CodeableConcept'    
-    embeds_many :author, class_name: 'FHIR::ContactDetail'    
-    embeds_many :editor, class_name: 'FHIR::ContactDetail'    
-    embeds_many :reviewer, class_name: 'FHIR::ContactDetail'    
-    embeds_many :endorser, class_name: 'FHIR::ContactDetail'    
-    embeds_many :relatedArtifact, class_name: 'FHIR::RelatedArtifact'    
-    embeds_many :parameter, class_name: 'FHIR::ParameterDefinition'    
-    embeds_many :dataRequirement, class_name: 'FHIR::DataRequirement'    
-    embeds_many :content, class_name: 'FHIR::Attachment'    
-    
+    embeds_one :url, class_name: 'FHIR::PrimitiveUri'
+    embeds_many :identifier, class_name: 'FHIR::Identifier'
+    embeds_one :version, class_name: 'FHIR::PrimitiveString'
+    embeds_one :name, class_name: 'FHIR::PrimitiveString'
+    embeds_one :title, class_name: 'FHIR::PrimitiveString'
+    embeds_one :subtitle, class_name: 'FHIR::PrimitiveString'
+    embeds_one :status, class_name: 'FHIR::PublicationStatus'
+    embeds_one :experimental, class_name: 'FHIR::PrimitiveBoolean'
+    embeds_one :type, class_name: 'FHIR::CodeableConcept'
+    embeds_one :subjectCodeableConcept, class_name: 'FHIR::CodeableConcept'
+    embeds_one :subjectReference, class_name: 'FHIR::Reference'
+    embeds_one :date, class_name: 'FHIR::PrimitiveDateTime'
+    embeds_one :publisher, class_name: 'FHIR::PrimitiveString'
+    embeds_many :contact, class_name: 'FHIR::ContactDetail'
+    embeds_one :description, class_name: 'FHIR::PrimitiveMarkdown'
+    embeds_many :useContext, class_name: 'FHIR::UsageContext'
+    embeds_many :jurisdiction, class_name: 'FHIR::CodeableConcept'
+    embeds_one :purpose, class_name: 'FHIR::PrimitiveMarkdown'
+    embeds_one :usage, class_name: 'FHIR::PrimitiveString'
+    embeds_one :copyright, class_name: 'FHIR::PrimitiveMarkdown'
+    embeds_one :approvalDate, class_name: 'FHIR::PrimitiveDate'
+    embeds_one :lastReviewDate, class_name: 'FHIR::PrimitiveDate'
+    embeds_one :effectivePeriod, class_name: 'FHIR::Period'
+    embeds_many :topic, class_name: 'FHIR::CodeableConcept'
+    embeds_many :author, class_name: 'FHIR::ContactDetail'
+    embeds_many :editor, class_name: 'FHIR::ContactDetail'
+    embeds_many :reviewer, class_name: 'FHIR::ContactDetail'
+    embeds_many :endorser, class_name: 'FHIR::ContactDetail'
+    embeds_many :relatedArtifact, class_name: 'FHIR::RelatedArtifact'
+    embeds_many :parameter, class_name: 'FHIR::ParameterDefinition'
+    embeds_many :dataRequirement, class_name: 'FHIR::DataRequirement'
+    embeds_many :content, class_name: 'FHIR::Attachment'
+
+    def create_data_elements(fhir_value_sets)
+      dataRequirement.map do |dataReq|
```

Comment:
This file is autogenerated so create_data_elements will be overridden

Reply:
ugh...that is a shame.  back to the drawing board on this one